pkg/profilecreator/profilecreator.go

```diff
@@ -229,22 +229,43 @@ func (ghwHandler GHWHandler) CPU() (*cpu.Info, error) {
 	return ghw.CPU(ghwHandler.snapShotOptions)
 }
 
-// Topology returns a TopologyInfo struct that contains information about the Topology on the host system
-func (ghwHandler GHWHandler) Topology() (*topology.Info, error) {
-	return ghw.Topology(ghwHandler.snapShotOptions)
+// SortedTopology returns a TopologyInfo struct that contains information about the Topology on the host system
+func (ghwHandler GHWHandler) SortedTopology() (*topology.Info, error) {
+	topologyInfo, err := ghw.Topology(ghwHandler.snapShotOptions)
+	if err != nil {
+		return nil, fmt.Errorf("Error obtaining Topology Info from GHW snapshot: %v", err)
+	}
+	// Sorting the NUMA nodes
```

Comment:
redundant comment (same below for line 243 and 249)

Reply:
Ack